testResults5.js

```diff
@@ -0,0 +1,250 @@
+        suma += i
+    }
+    return suma
+};
+let arr2 = [23, 35, 63, 33, 89, 92, 42, 7];
+console.log(sum(arr2)); 
+
+// Zadatak 3
+let niz2 = [1, 3, 5, 7];
+function proizvod (arr) {
+    let mult = 1
+    for (const i of arr) {
+        mult *= i
+    }
+    return mult;
+};
+console.log(proizvod(niz2));
+
+// Zadatak 4
+let k = 10;
+let b = 35;
+function neparni (x, y) {
+    let niz3 = [];
+    for (i = x; i <= y; i++){
+        if (i % 2 != 0) {
+            niz3.push(i)
+        }
+    }
+    return niz3
+};
+console.log(neparni(k, b));
+
+// Zadatak 5
+let niz3 = [23, 35, 63, 33, 89, 92, 42, 7];
+function sredVr (arr) {
+    let srednjaVrEl = [];
+    for (const a of arr){
+        srednjaVrEl.push(a / 2)
+    }
+    return srednjaVrEl
+}
+console.log(sredVr(niz3));
+
+// Zadatak 6
+let niz4 = [7, 35, 53, 63, 89, 92];
+function granica (arr, a, b) {
+    for (const x of arr) {
+        ( x >= a && x <= b ) ? console.log(true) : console.log(false)
+    }
+}
+let bool = granica(niz4, Math.min(...niz4), Math.max(...niz4))
+console.log(bool);
+
+// Zadatak 7
+let niz5 = [23, 35, 63, 33, 89, 92, 42, 7];
+function granica2 (arr, a, b) {
+    for (const x of arr) {
+        ( x >= a && x <= b ) ? console.log(true) : console.log(false)
+        }
+}
+let bool2 = granica2(niz5, Math.min(...niz5), Math.max(...niz5))
+console.log(bool2);
+
+// Zadatak 8
+let n = 10;
+function desetka (broj) {
+    let arr = [];
+    for (i = n; arr.length < 10; i++){
+        if (i % 2 ===0) {
+            arr.push(i)
+        }
+    }
+    return arr
+};
+console.log(desetka(n));
+
+// Zadatak 9
+let array = [23, 7, 35, 63, 7, 89, 92, 42, 7];
+let a = 7;
+function check (arr, i) {
+    return arr.filter(el => el === i).length
+};
+console.log(check(array, a));
+
+// Zadatak 10
+let array2 = [23, 7, 35, 63, 7, 89, 92, 42, 7];
+let a3 = 7;
+function check (arr, i) {
+    let newArr = []
+    for (const x of arr) {
+        if (x % i === 0) {
+            newArr.push(x)
+        }
+    }
+    return newArr.length
+};
+console.log(check(array2, a3));
+
+// Zadatak 11
+let niz6 = [23, 7, 35, 63, 89, 92, 42];
+let najmanji = (arr) => arr.indexOf(Math.min(...arr))
+console.log(najmanji(niz6));
+
+// Zadatak 12
+let niz2d = [
+    [1, 3, 5, 7, 9], 
+    [2, 4, 6, 8, 10]
+];
+function podelementi (arr) {
+    for(const i of arr){
+        for(const x of i)
+        console.log(x)
+    }
+};
+podelementi(niz2d);
+
+// Zadatak 13
+let niz2d2 = [
+    [1, 3, 5, 7, 9], 
+    [2, 4, 6, 8, 10]
+];
+function suma (arr) {
+    let sum = 0;
+    for(const i of arr){
+        for(const x of i)
+        sum += x
+    }
+    return sum
+};
+console.log(suma(niz2d2));
+
+// Zadatak 14
+let niz12 = [
+	[1, 2, 3, 4],
+	[5, 6, 7, 8],
+    [9, 10, 11, 12],
+	[13, 14, 15, 16]
+];
+let niz13 = ['down', 'down', 'right', 'down', 'right'];
+function kretanje (niz12, niz13) {
+    let i = 0;
+    let j = 0;
+    for (const korak of niz13) {
+        if (korak === 'down') {
```

Comment:
change this to switch... never use else if, else if... that is a place for switch